hbase-common/src/main/java/org/apache/hadoop/hbase/CellComparatorImpl.java

```diff
@@ -51,35 +53,38 @@
    */
   public static final CellComparatorImpl COMPARATOR = new CellComparatorImpl();
 
+  private static final ContiguousCellFormatComparator contiguousCellComparator =
```

Comment:
the term Contiguous was added in terms of the Kv format. In case of cells that are encoder based it is alwasy from left to right but the format may not be KV based. Hence I went with the word 'contiguous'.

Reply:
Haven't checked but might be worth adding this note on what contiguous means to your marker interface....